README.md

```diff
@@ -1 +1,87 @@
-# sound-analyser-for-LEDs
\ No newline at end of file
+
+# sound-analyser-for-LEDs
+
+## Table of Contents
+ - [Introduction](#introduction) 
+ - [Features](#features) 
```

Comment:
Can you add also depencencies list please?

Reply:
Sure